templates/donor-registered.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends renderHtml ? '_base.html.twig' : '_base.text.twig' %}
+
+{% block content %}
+  <p>Dear {{ donorFirstName }},</p>
+
+  <p>You are now registered for the Big Give with the username {{ donorUsername }}. This will make donating to your chosen campaigns much easier in the future.</p>
```

Comment:
I think we should be saying "email address" since we never scoped collecting a separate username and I'm not convinced having one would add anything.